### ISSUE TITLE
Handling Citations

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#914]
We should decide how we want ASPECT to be cited and advertise this decision. Inspired by SAGA (https://geodynamics.org/cig/projects/saga/).

How to cite:
- ASPECT paper
- ASPECT manual?
- DOI for each version? (problem: getting credit, splitting citations, no google scholar tracking ...) 
- refer to other papers (free surface, melt migration, tracers)
- maybe even include other packages like deal.II? Trilinos?? etc.
- suggest acknowledgement section in each paper (like SAGA suggests: "We thank the Computational Infrastructure for Geodynamics (geodynamics.org) which is funded by the National Science Foundation under award NSF-0949446.")

How to advertise:
- ASPECT manual
- CITATION file in repo
- website: http://aspect.dealii.org/publications.html
- use SAGA website?
- create output/citation during a run (to include version numbers, git hash, and modules used), link to SAGA website with correct boxes checked? Local website with javascript?

Steps:
1. Discuss how we want ASPECT to be cited
2. Discuss with SAGA group: @lkellogg @ljhwang 
3. Decide whether to use the SAGA website
4. Implement actions

**Comment by bangerth** [comment]
There is already section 1.1 "Referencing ASPECT" in the manual. It should be relatively easy to point users to it in the ASPECT start-up message at the top of the log file of every run.

**Comment by ljhwang** [comment]
Yes, ASPECT does a good job as it gives the bibtek.  The next step would be information on versioning which may include DOIs

Best,
-Lorraine

*****************************
Lorraine Hwang, Ph.D.
Associate Director, CIG
530.752.3656

**Comment by tjhei** [comment]
> There is already section 1.1 "Referencing ASPECT" in the manual.

Yes, I know. I am just listing all current and potential locations to make sure that we update all of them.

**Comment by ljhwang** [comment]
@tjhei DOIs may not be used by google but they are used by most of the indexers and is something they are used to handling.  What would google scholar need? How do people use it in context of discovery and credit for code?
